### ISSUE TITLE
Crash loop and new AccessDeniedException

### ISSUE THREAD

**Issue by geekdave** [danielfm/aws-limits-exporter#11]
Thanks so much for your work on this exporter.  We've had it running reliably for months without issue.  Suddenly this morning we started getting a crash loop.  Tried upgrading to version `0.3.1` which seemed to have a fix for a nil pointer error but same problem.

```
goroutine 20 [running]:
github.com/danielfm/aws-limits-exporter/core.(*SupportExporter).Describe(0xc0001e3800, 0xc0000d05a0)
	/build/core/support.go:236 +0x5a3
github.com/prometheus/client_golang/prometheus.(*Registry).Register.func1(0xc14680, 0xc0001e3800, 0xc0000d05a0)
	/go/pkg/mod/github.com/prometheus/client_golang@v1.2.1/prometheus/registry.go:274 +0x3b
created by github.com/prometheus/client_golang/prometheus.(*Registry).Register
	/go/pkg/mod/github.com/prometheus/client_golang@v1.2.1/prometheus/registry.go:273 +0x154
I1021 17:51:31.346449       1 main.go:26] AWS Limits Exporter v0.3.1 started.
I1021 17:51:31.347550       1 support.go:172] Refreshing Trusted Advisor check 'fW7HH0l7J9'...
E1021 17:51:31.455144       1 support.go:233] Cannot retrieve Trusted Advisor check results data: AccessDeniedException:
	status code: 400, request id: a980614c-fee7-4433-a949-085b8d5755e2
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x92a213]

goroutine 9 [running]:
github.com/danielfm/aws-limits-exporter/core.(*SupportExporter).Describe(0xc00019ced0, 0xc000060de0)
	/build/core/support.go:236 +0x5a3
github.com/prometheus/client_golang/prometheus.(*Registry).Register.func1(0xc14680, 0xc00019ced0, 0xc000060de0)
	/go/pkg/mod/github.com/prometheus/client_golang@v1.2.1/prometheus/registry.go:274 +0x3b
created by github.com/prometheus/client_golang/prometheus.(*Registry).Register
	/go/pkg/mod/github.com/prometheus/client_golang@v1.2.1/prometheus/registry.go:273 +0x154
```

Not sure if it's an AWS problem about why we suddenly started getting the `AccessDeniedException`.  No changes to our IAM user/policy.

**Comment by geekdave** [comment]
The access denied error resolved itself so it may have been an issue with the trusted advisor API on the AWS side.  But perhaps the error could be caught more gracefully so as not to crash the process?

![prom](https://user-images.githubusercontent.com/1438478/96772863-9888f000-1398-11eb-9960-49bc2220aeb3.jpg)

**Comment by danielfm** [comment]
This might be fixed in #12, since the `fW7HH0l7J9` check was deprecated. I'll close this for now, feel free to reopen if you still experience issues.
